docs/source/distillation_quantization.md

```diff
@@ -66,7 +66,6 @@ def training_func_for_nc(model):
 In this case, the launcher code is like the following:
 
 ```python
-from neural_compressor.experimental import common, Distillation, Quantization
```

Comment:
This doc should be removed, https://github.com/intel/neural-compressor/pull/2206/files.

Reply:
ok